data/Dockerfiles/acme/docker-entrypoint.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+ACME_BASE=/var/lib/acme
+mkdir -p ${ACME_BASE}/acme/private
+
+restart_containers(){
+	for container in $*; do
+		curl -X POST \
+			--unix-socket /var/run/docker.sock \
+			"http:/containers/${container}/restart"
+	done
+}
+
+while true; do
+
+	acme-client \
+		-v -b -N -n \
+		-f ${ACME_BASE}/acme/private/account.key \
+		-k ${ACME_BASE}/acme/private/privkey.pem \
+		-c ${ACME_BASE}/acme \
+		${MAILCOW_HOSTNAME} autoconfig.${MAILCOW_HOSTNAME#*.} autoconfig.${MAILCOW_HOSTNAME#*.} ${ADDITIONAL_SAN}
```

Comment:
Oh. Didn't notice this. The dev differs a bit though: https://github.com/mailcow/mailcow-dockerized/blob/dev/data/Dockerfiles/acme/docker-entrypoint.sh

This code checks for autodiscover/autoconfig.{each_domain} and tries to create a certificate.

Yes, a SRV works fine, too. Should be part of the docs.